test/oauth-authorization-test.js

```diff
@@ -37,6 +37,20 @@ var assert = require("assert"),
     accessToken = oauthutil.accessToken;
 
 var ignore = function(err) {};
+
+var browserClose=function(br){
+	Step(
+		function(){
+			br.on("closed",this);
+			br.window.close();
+		},
+		function(br){
+			//browser is closed;
+		}
```

Comment:
Synchronized closing. I think vows doesn't like a topic to end and have some callback coming back after the topic is closed. 
#### I'm open for a better solution. 
If I remember correctly `br.window.close(function(){})` didn't do the trick.